android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDatabaseMigration.java

```diff
@@ -1,48 +1,31 @@
 package org.fossasia.openevent.dbutils;
 
 import io.realm.DynamicRealm;
+import io.realm.RealmConfiguration;
 import io.realm.RealmMigration;
+import io.realm.RealmSchema;
 
 public class RealmDatabaseMigration implements RealmMigration {
 
     public static final long DB_VERSION = 0;
 
-    /**
-     * Defines migration strategy of Realm Database
-     * Refer to https://realm.io/docs/java/latest/#migrations for more info
-     * @param dynamicRealm realm schema instance
-     * @param oldVersion The existing database version
-     * @param newVersion The database version migration will happen to
-     */
     @Override
-    public void migrate(DynamicRealm dynamicRealm, long oldVersion, long newVersion) {
-
-        // Add schema increments and change methods when updating models
-
-        /* Example migration :
+    public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
+        //updating schema version
+        RealmConfiguration config = new RealmConfiguration.Builder()
+                .schemaVersion(2)
+                .build();
```

Comment:
This is unneeded. Look at OpenEventApplcation class, the config is used there. So I guess only DB_VERSION needs to be incremented